guides/project-service-management-guide.md

```diff
@@ -405,6 +409,25 @@ parameters to `spk service create --helm-config-git`:
 
 `https://dev.azure.com/fabrikam/fabrikam-project/_git/fabrikam-helm-charts`
 
+#### Passing Variables as Dockerfile Build Arguments
+
+If you want to pass in build arguments during the Dockerfile build process, you
+can inject them using the `--service-build-vg` and `--service-build-variables`
+arguments, which will take in multiple variable groups and variables,
+respectively.
+
+```
+spk service create
+  --helm-config-git https://dev.azure.com/fabrikam/fabrikam-project/_git/fabrikam-app \
+  --helm-config-branch master \
+  --helm-path /charts/fabrikam \
+  --service-build-vg fabrikam-vg \
+  --service-build-variables FOO,BAR
```

Comment:
Why comma separated instead of something more akin to docker where you use the same CLI argument multiple times?

Reply:
There is an example in the project-service-management-guide.md.

Reply:
To remain consistent with how `spk` takes in arguments. We are also doing the same in other areas within `spk`. For example: https://github.com/microsoft/bedrock-cli/blob/60c89e1d7c09ae55bb622359baa3a6bbba2824a0/src/commands/service/create.decorator.json#L57.

Reply:
@mtarng @evanlouie can probably provide a little more context.